lib/traject/config/folio_config.rb

```diff
@@ -2560,52 +2551,6 @@ def accumulate_summary_struct_fields(matching_fields, tag, label, marc, accumula
   end
 end
 
-# We update item_display once we have crez info
-to_field 'item_display_struct' do |_record, _accumulator, context|
-  course_reserves = context.clipboard[:crez_data]
-  next if course_reserves.empty?
-
-  context.output_hash['item_display_struct'].map! do |item_display_value|
-    row = course_reserves.reverse.find { |r| r[:barcode].strip == item_display_value[:barcode]&.strip }
-
-    if row
-      rez_desk = row[:rez_desk] || ''
-      loan_period = LOAN_CODE_2_USER_STR[row[:loan_period]] || ''
-      course_id = row[:course_id] || ''
-
-      item_display_value.merge(
-        current_location: rez_desk,
-        course_id:,
-        reserve_desk: rez_desk,
-        loan_period:
-      )
-    else
-      item_display_value
-    end
-  end.flatten!
-end
-
-to_field 'building_facet' do |_record, _accumulator, context|
-  course_reserves = context.clipboard[:crez_data]
-  next if course_reserves.empty?
-
-  new_building_facet_vals = context.output_hash['item_display_struct'].map do |item_display_value|
-    barcode = item_display_value[:barcode]&.strip
-    reserves_for_item = course_reserves.select { |row| row[:barcode].strip == barcode }.first
-
-    if reserves_for_item && REZ_DESK_2_BLDG_FACET[reserves_for_item[:rez_desk]]
-      REZ_DESK_2_BLDG_FACET[reserves_for_item[:rez_desk]]
-    else
-      # This is not dissimilar to the original building_facet mapping:
-      # Try the current location first, in case it has an overridden library, and then
-      # fall back on the library code.
-      library_map[item_display_value[:current_location]] || library_map[item_display_value[:library]]
-    end
-  end
-
-  context.output_hash['building_facet'] = new_building_facet_vals.uniq if new_building_facet_vals.any?
-end
-
```

Comment:
Just for my understanding: are we allowed to remove these because we look for and use temp locations at the time we construct `item_display_struct` and `building_facet`, and thus have no need to revisit/alter them? What about stuff like `course_id` and `loan_period`? I don't see us adding that `item_display_struct` anywhere else.

Reply:
Right -- the FOLIO item record is actually right up-front.

Reply:
Oh, I see — and `courses_json_struct` handles the rest, I guess? cool.